Lombiq.UIKit.csproj

```diff
@@ -32,6 +32,7 @@
 
   <ItemGroup>
     <None Remove="node_modules\**" />
+    <None Remove="wwwroot\**" />
```

Comment:
Does this needs to be done in every project with a webroot folder?

Reply:
It's not mandatory - it was an opinionated choice to hide the generated assets. Should we not?

Reply:
I don't think so.